short/deploy-to-azure-from-github/README.md

```diff
@@ -0,0 +1,199 @@
+# Microsoft Student Summit
+## Launch into the Cloud with GitHub and Azure
+<hr>
```

Comment:
this is a left over tag that should be removed

Reply:
Done!